### ISSUE TITLE
Add non versioned urls

### ISSUE THREAD

**Issue by SimonCropp** [ravendb/docs#112]
So i want generic links for a given topic that will always give the current information

For example

unique-constraints the current urls are
http://ravendb.net/docs/article-page/2.5/csharp/server/extending/bundles/unique-constraints  
http://ravendb.net/docs/article-page/3.0/csharp/server/bundles/unique-constraints
I want a url like http://ravendb.net/docs/bundles/unique-constraints

upgrading to a new version 
http://ravendb.net/docs/article-page/2.5/csharp/server/administration/upgrade
http://ravendb.net/docs/article-page/3.0/csharp/server/installation/upgrading-to-new-version
I want a url like http://ravendb.net/docs/article-page/csharp/server/administration/upgrade

management studio
http://ravendb.net/docs/article-page/2.5/csharp/studio/index
http://ravendb.net/docs/article-page/3.0/csharp/studio/accessing-studio
I want a url like http://ravendb.net/docs/article-page/csharp/studio/index

Basically for anyone referring to the ravendb doco it is a constant process of manually searching and finding the new urls for each version.

ATM I think the only option given the current url structure is to give people urls like http://ravendb.net/search?q=unique-constraints

Note in all of the above examples the version switcher is broken

**Comment by ppekrol** [comment]
Hi Simon,

thanks for feedback. We will create a mechanism in version switcher that will redirect to search for a chosen version.

The generic urls will be applied with the next version of documentation.

**Comment by SimonCropp** [comment]
seems you release a new version. since this is now broken http://ravendb.net/search?q=IDocumentConversionListener

so I assume this one is now fixed?

**Comment by ppekrol** [comment]
Try:
http://ravendb.net/docs/search/latest/csharp?searchTerm=IDocumentConversionListener

**Comment by SimonCropp** [comment]
@ppekrol thanks. perhaps you should also put a redirect on the old one rather than giving a stack trace

**Comment by ppekrol** [comment]
Good call, thanks! Added support for legacy search.

**Comment by SimonCropp** [comment]
any update on this ?

**Comment by ppekrol** [comment]
Each version contains it's own unique features and we do not see a good way of doing this. But you can use search: http://ravendb.net/docs/search/latest/csharp?searchTerm=IDocumentConversionListener

**Comment by SimonCropp** [comment]
@ppekrol so u r saying no on should be deeplinking into any raven doco since that link will eventfully be out of date. isnt this a problem for every raven link on a site like stackoverflow? so u have a gradually decaying quality of referring links

**Comment by SimonCropp** [comment]
here are some examples

 * http://stackoverflow.com/questions/27022427/ravendb-load-related-data-not-working
 * http://stackoverflow.com/questions/34652296/ravendb-creating-static-index-and-query-on-dictionary/34677157
 * http://stackoverflow.com/questions/27045268/ravendb-issue-of-matching-with-search-and-wilcards

**Comment by ppekrol** [comment]
Links in those articles are valid, they point to an exact article in 2.5 documentation. We do not plan to change this.
